python/src/aiconfig/editor/client/src/components/ParametersRenderer.tsx

```diff
@@ -100,7 +101,7 @@ const ParameterInput = memo(function ParameterInput(props: {
         />
         <Textarea
           placeholder="Enter parameter value"
-          disabled={isReadonly}
+          disabled={readOnly}
```

Comment:
I personally don't really like the greyed out style, would rather change theming (probably globally is better since we'll probably do this for text components in prompt input, prompt output, model settings, prompt params as well

cc @zakariassoul what do you think about the greyed out styling?

Not a blocker, but think it should be P1
cc @rholinshead can you give Ankush code pointers on this after bug-bash/MVP is released?

Reply:
+1, I do think the style is generally pretty dark and a bit hard to see things